public/index.php

```diff
@@ -15,7 +15,9 @@
 use Symfony\Component\ErrorHandler\Debug;
 use Symfony\Component\HttpFoundation\Request;
 
-\defined('SULU_MAINTENANCE') || \define('SULU_MAINTENANCE', \getenv('SULU_MAINTENANCE') ?: false);
+if (!\defined('SULU_MAINTENANCE')) {
+    \define('SULU_MAINTENANCE', \getenv('SULU_MAINTENANCE'));
```

Comment:
`getenv` returns false by default.
https://www.php.net/manual/en/function.getenv.php

Reply:
Should it be `(bool)\getenv('SULU_MAINTENANCE')` instead?

Reply:
at the end currently it is set if not empty it will be in Maintainance mode. all other things would be currently a bc break.